### ISSUE TITLE
Make setter names consistent

### ISSUE THREAD

**Issue by vyasr** [glotzerlab/coxeter#114]
The setters for Python properties should consistently either use the name of the property or just `value` as the argument. This will improve code consistency and readability.

**Comment by vyasr** [comment]
@Tobias-Dwyer I'm assigning this task to you as a follow-up to your open PRs. You can work on this once #105 is done. I'm moving this to a separate issue rather than your open PRs because there are a large number of properties and what is needed here is a thorough review, not just fixing the properties that you're currently adding setters for.
